wee_slack.py

```diff
@@ -2240,6 +2240,8 @@ def __init__(self, eventrouter, parent_message):
         self.members = self.parent_message.channel.members
         self.team = self.parent_message.team
         self.last_line_from = None
+        self.last_read = self.parent_message.message_json.get("last_read", SlackTS())
```

Comment:
This attribute on the parent message is only updated when the channel history is reloaded as far as I can see. That means that if you close the thread buffer and open it again, the last read indicator will be where it was the last time the channel history was loaded.

Reply:
OK, I think I solved this one.